llm/gguf.go

```diff
@@ -38,21 +31,29 @@ type ContainerGGUF struct {
 	}
 }
 
-func (c *ContainerGGUF) Name() string {
+func (c *containerGGUF) Name() string {
```

Comment:
Can we move this to a `gguf` package in the future – would be better if this `llm` package was focused on the runner

Reply:
I intended the `llm` package to encapsulate any large language model related code including the model (parsing, conversion, writing) in addition to running